cmd/state/stateless/state_snapshot.go

```diff
@@ -32,11 +32,14 @@ func constructSnapshot(ethDb ethdb.Database, blockNum uint64) {
 	var startKey [32]byte
 	txDisk, err := diskDb.Begin(true)
 	check(err)
-	bDisk, err := txDisk.CreateBucket(dbutils.AccountsBucket, true)
+	bDisk, err := txDisk.CreateBucket(dbutils.CurrentStateBucket, true)
 	check(err)
 	count := 0
-	err = ethDb.WalkAsOf(dbutils.AccountsBucket, dbutils.AccountsHistoryBucket, startKey[:], 0, blockNum+1,
+	err = ethDb.WalkAsOf(dbutils.CurrentStateBucket, dbutils.AccountsHistoryBucket, startKey[:], 0, blockNum+1,
 		func(key []byte, value []byte) (bool, error) {
+			if len(key) != 32 {
+				return false, nil
```

Comment:
here must be true, otherwise, code walk over 1 account only - then stop (instead of walk over all accounts)

Reply:
I will fix it in branch `add_incarnation_to_acc_root_in_ih `